packages/cli/src/commands/init/init.js

```diff
@@ -162,7 +162,7 @@ async function installDependencies({
   });
 
   if (process.platform === 'darwin') {
-    await installPods({projectName, loader});
+    await installPods({projectName, loader, shouldUpdatePods: false});
```

Comment:
Could we have an option to enable this for template configs - there's an issue with any custom templates that use pod specs where the template will fail to init because it could not find the pod specs (because a repo update is required)

See: https://github.com/invertase/react-native-firebase/issues/2351

Reply:
`installPods` checks if the template has a Podfile, so it's gonna work in your template as well

Reply:
Strangely, this did not work for a react-native-firebase v6 user just now, in that they had stale cocoapods repo while the react-native-firebase v6 starter template was looking for some pretty recent pods, and pod install failed because pod update needed to run. A manual run of pod update for the user resolved it so I think somewhere in here is the root cause but I'm not sure what the interaction is.

I believe this is the template Podfile: https://github.com/invertase/react-native-firebase/blob/master/packages/template/project/ios/Podfile

Reply:
got a repro and logged it up here https://github.com/react-native-community/cli/issues/752